### ISSUE TITLE
Warning when exiting on Linux

### ISSUE THREAD

**Issue by parkerlreed** [ruffle-rs/ruffle#3028]
**Describe the bug**

Warning is printed when exiting the program. Doesn't affect functionality

```
[parker@stealth ~]$ ruffle kittencannon.swf 
[2021-02-01T20:13:05Z ERROR gpu_descriptor::allocator] `DescriptorAllocator` is dropped while some descriptor sets were not deallocated 
```

**Expected behavior**

No warning

**Is the problem with the Ruffle desktop app, extension, or self-hosted version?**

Desktop 

**What platform are you using?**

Arch Linux
AMDGPU Mesa Vukan
ruffle master (00e312c5f2458e5c9fbf617148f9b78b6270bc48)

**Comment by Herschel** [comment]
This is a known issue with wgpu vulkan backend and will hopefully go away when we bump versions.

**Comment by relrelb** [comment]
Fixed by #3066.
